tests/test_recipient_validation.py

```diff
@@ -350,18 +311,18 @@ def test_validates_against_guestlist_of_email_addresses(email_address):
 
 
 @pytest.mark.parametrize("phone_number, expected_formatted", [
-    ('07900900123', '07900 900123'),  # UK
-    ('+44(0)7900900123', '07900 900123'),  # UK
-    ('447900900123', '07900 900123'),  # UK
-    ('20-12-1234-1234', '+20 121 234 1234'),  # Egypt
-    ('00201212341234', '+20 121 234 1234'),  # Egypt
-    ('1664 0000000', '+1 664-000-0000'),  # Montserrat
-    ('7 499 1231212', '+7 499 123-12-12'),  # Moscow (Russia)
-    ('1-202-555-0104', '+1 202-555-0104'),  # Washington DC (USA)
+    ('+4407900900123', '+44 7900 900123'),  # UK
+    ('+44(0)7900900123', '+44 7900 900123'),  # UK
+    ('+447900900123', '+44 7900 900123'),  # UK
+    ('+20-12-1234-1234', '+20 121 234 1234'),  # Egypt
+    ('+201212341234', '+20 121 234 1234'),  # Egypt
+    ('+1 664 491-3434', '(664) 491-3434'),  # Montserrat
```

Comment:
Hmmm, is this actually an international number for a US caller? If so, how should we be handling it?

Perhaps this is also a TODO as part of https://github.com/GSA/notifications-api/issues/135

Reply:
So... whether a number within the [`+1` country code](https://en.wikipedia.org/wiki/North_American_Numbering_Plan) is international does not appear to be an answerable question. The `phonenumbers` library assigns them all with the `1` country code. Whether they're "international" for a given phone call appears to be entirely dependent on the caller's phone plan. 

Beyond that, it appears that the only way to figure out if _we_ need to pay more for a given SMS is to send it...

> The price for sending SMS messages varies between countries, regions, and in some cases, between carriers in the same country or region based on the telecom carrier of the destination phone number. The prices below are provided for guidance only, and change frequently. Once you’ve sent some messages, use the daily usage reports to get the exact pricing per message in USD.

source: https://aws.amazon.com/sns/sms-pricing/